### PR TITLE
CHE-1210. Block content assist until project is resolved

### PR DESCRIPTION
**Pull request by RomanNikitenko**
Signed-off-by: Roman Nikitenko rnikitenko@codenvy.com

### CODE REVIEW

**Comment by vparfonov** [comment]
ok

File: plugins/plugin-maven/che-plugin-maven-ide/src/main/java/org/eclipse/che/plugin/maven/client/project/ResolvingMavenProjectStateHolder.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+import java.util.HashSet;
+
+import static org.eclipse.che.ide.project.ResolvingProjectStateHolder.ResolvingProjectState.IN_PROGRESS;
+import static org.eclipse.che.ide.project.ResolvingProjectStateHolder.ResolvingProjectState.NOT_RESOLVED;
+import static org.eclipse.che.ide.project.ResolvingProjectStateHolder.ResolvingProjectState.RESOLVED;
+import static org.eclipse.che.plugin.maven.shared.MavenAttributes.MAVEN_CHANEL_NAME;
+import static org.eclipse.che.plugin.maven.shared.MavenAttributes.MAVEN_ID;
+import static org.eclipse.che.plugin.maven.shared.MessageType.START_STOP;
+
+@Singleton
```
Comment (vparfonov) [inline plugins/plugin-maven/che-plugin-maven-ide/src/main/java/org/eclipse/che/plugin/maven/client/project/ResolvingMavenProjectStateHolder.java:53]: Javadoc

Comment (RomanNikitenko) [inline plugins/plugin-maven/che-plugin-maven-ide/src/main/java/org/eclipse/che/plugin/maven/client/project/ResolvingMavenProjectStateHolder.java:53]: @vparfonov done!

**Comment by vparfonov** [comment]
ok

**Pull request merged**